cpp/ql/src/Security/CWE/CWE-497/PotentiallyExposedSystemData.qhelp

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>Exposing system data or debugging information may help an adversary to learn about the system and form an attack plan.  An attacker can use error messages that reveal technologies, operating systems, and product versions to tune their attack against known vulnerabilities in these technologies.</p>
```

Comment:
I'd make the same changes to to this qhelp file that I suggested for the other qhelp file `adversary` -> `malicious user`, remove the `to` in `may help an adversary to learn about`, and possibly find a synonym for `technologies`.
I am a bit confused because the content for both seems to be exactly the same 😕

Reply:
Done.

We've essentially forked one query into two variations here, and ended up with two versions of the `qhelp` as well.  There are slight differences reflecting the difference in focus between the two queries, most of the text remains identical.

Reply:
Thanks for explaining, it now makes more sense to me 🙂